src/Client/GameClient.cpp

```diff
@@ -103,16 +104,32 @@ void GameClient::renderPlayers() {
 
 void GameClient::update(GameMsgPlaying updateObjects) { //ToDo por ahora solo actualizamos las cosas de los jugadores ya que no hay colisiones y esas cosas
```

Comment:
Change this todo as you are fixing it now

Reply:
Comenta en español gilazo